src/character.h

```diff
@@ -2394,12 +2394,12 @@ class Character : public Creature, public visitable
                 const nutrients *nutrient = nullptr ) const;
         /** Handles the effects of consuming an item */
         bool consume_effects( item &food );
-        /** Check character's capability of consumption overall */
+        /** Check whether the character can consume this very item */
+        bool can_consume_as_is( const item &it ) const;
+        /** Check whether the character can consume this item or any of its contents */
```

Comment:
There's no real need to move this around, it's probably best to avoid churn.

Reply:
Moving declarations around was an afterthought: really, I wanted to clarify the comment, as it implied the previous (incorrect) behavior was the intention. I can revert either or both if you insist.

Reply:
I guess I didn't look closely enough, my bad.
If you feel it's worth the effort to move it back to the original location with an update comment, do so, otherwise it's fine because you're actually making changes.

Reply:
I'll have it stay then - these two functions now match the order they're declared in the `.cpp`. Thank you for your time!